### PR TITLE
Depl-165154 install-web.sh sets incorrect ownership for the conf/Catalina/<websitename> directory and its content

### PR DESCRIPTION
**Pull request by KupchinskiyVasiliy**

### CODE REVIEW

File: setup/initial/aws-setup-web-server.sh [hunk 1]
```diff
@@ -38,5 +38,11 @@ systemctl enable iptables
 iptables -t nat -I PREROUTING -p tcp --dport 80 -j REDIRECT --to-port 8080
 service iptables save
 
+# Install yum plugin
+yum install -y yum-plugin-post-transaction-actions
+
+# Create yum post-action for restore permissions after Tomcat package install/update
```
Comment (asmoiseenko) [inline setup/initial/aws-setup-web-server.sh:44]: to restore

File: setup/initial/README.md [hunk 2]
```diff
@@ -212,3 +213,6 @@ This IAM Role should contains next permissions:
     ```
 
 5. Replace `[placeholder for Oracle SID]` and `[placeholder for Oracle host]` with correct values.
+
+## Upgrade Tomcat on non-AWS environment
+1. After upgrade Tomcat on non-AWS environment run `./config-tomcat-security.sh` for restore Tomcat files and directories correct permissions.
```
Comment (asmoiseenko) [inline setup/initial/README.md:218]: After tomcat upgrade

File: setup/templates/yum/post-actions/tomcat.action [hunk 3]
```diff
@@ -0,0 +1,2 @@
+tomcat*:install:"$SCRIPTS_PATH/config-tomcat-security.sh"
```
Comment (asmoiseenko) [inline setup/templates/yum/post-actions/tomcat.action:1]: how it will know the value of $SCRIPT_PATH here?

**Review by asmoiseenko — CHANGES_REQUESTED** [review]

File: setup/templates/yum/post-actions/tomcat.action (hunk 3 above)
Comment (KupchinskiyVasiliy) [reply setup/templates/yum/post-actions/tomcat.action:1]: Will be resolved through envsubst (check aws-setup-web-server.sh)

File: setup/initial/aws-setup-web-server.sh (hunk 1 above)
Comment (KupchinskiyVasiliy) [reply setup/initial/aws-setup-web-server.sh]: Done.

File: setup/initial/README.md (hunk 2 above)
Comment (KupchinskiyVasiliy) [reply setup/initial/README.md]: Done.

**Review by asmoiseenko — APPROVED** [review]

**Pull request merged**